app/assets/javascripts/admin/controllers/admin-web-hooks-show.js.es6

```diff
@@ -80,12 +80,24 @@ export default Ember.Controller.extend({
   actions: {
     save() {
       this.set('saved', false);
-
+      const url = extractDomainFromUrl(this.get('model.payload_url'));
       const model = this.get('model');
-      return model.save().then(() => {
-        this.set('saved', true);
-        this.get('controllers.adminWebHooks').get('model').addObject(model);
-      }).catch(popupAjaxError);
+      const saveWebHook = () => {
+        return model.save().then(() => {
+          this.set('saved', true);
+          this.get('controllers.adminWebHooks').get('model').addObject(model);
+        }).catch(popupAjaxError);
+      };
+
+      if (url.indexOf('localhost') !== -1 || url.match(/192\.168\.\d+\.\d+/) || url.match(/127\.\d+\.\d+\.\d+/) || url === Discourse.BaseUrl) {
```

Comment:
hmmm if the domain is extracted from the url, why will `localhost` or the `IP`s be checked for here?

Can you post some examples of the URLs being checked here? I'm working on smth now so I'll be hard for me to pull down the branch and check